docs/content/getting-started/kubernetes.md

```diff
@@ -0,0 +1,387 @@
+---
+title: "Kubernetes and Traefik Quick Start"
+description: "Deploy Traefik in Kubernetes using Helm and expose your first service"
+---
+
+# Getting Started with Kubernetes and Traefik
+
+Kubernetes is a first-class citizen in Traefik, offering native support for Kubernetes resources and the latest Kubernetes standards. 
+Whether you're using Traefik's [IngressRoute CRD](../reference/routing-configuration/kubernetes/crd/http/ingressroute.md) or the Kubernetes Gateway API, 
```

Comment:
Here I think we should add a link to the corresponding documentation.

Reply:
Also I am thinking that even if the Ingress spec is frozen, we should mention it.